### DISCUSSION TITLE
Semantic-release will check branch name for release range check?

### DISCUSSION THREAD

**Discussion by luisxiaomai** [semantic-release/semantic-release#2457] [Release workflow] [answered]
## Current behavior
Currently master branch is 2.x.x version. There is another branch names "2.6" not version which will publish 1.x.x version. But it will failed with below error, why it checked publish version should >=2.6.0? 
```
EINVALIDNEXTVERSION The release `1.3.5` on branch `2.6` cannot be published as it is out of range.
Based on the releases published on other branches, only versions within the range >=2.6.0 <2.0.0 can be published from branch 2.6.
```

## Expected behavior
"2.6" branch can publish version <2.0.0, not limited >=2.6.0, version 1.3.5 should be published from branch "2.6"

Appreciate any help on this, thanks.

**Comment by travi** [comment, marked as answer]
please see https://semantic-release.gitbook.io/semantic-release/usage/workflow-configuration#maintenance-branches
